### ISSUE TITLE
Can't download CSV from Tensorboard GUI

### ISSUE THREAD

**Issue by cfRod** [tensorflow/tensorboard#6551]
Consider Stack Overflow for getting support using TensorBoard—they have
a larger community with better searchability:

https://stackoverflow.com/questions/tagged/tensorboard

Do not use this template for for setup, installation, or configuration
issues. Instead, use the “installation problem” issue template:

https://github.com/tensorflow/tensorboard/issues/new?template=installation_problem.md

To report a problem with TensorBoard itself, please fill out the
remainder of this template.

## Environment information (required)

Please run `diagnose_tensorboard.py` (link below) in the same
environment from which you normally run TensorFlow/TensorBoard, and
paste the output here:
### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version df7af2c6fc0e4c4a5b47aeae078bc7ad95777ffa

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=9, micro=4, releaselevel='final', serial=0)
INFO: os.name: posix
INFO: os.uname(): posix.uname_result(sysname='Darwin', nodename='FVFF77UXQ05N', release='22.5.0', version='Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:19 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T8103', machine='arm64')
INFO: sys.getwindowsversion(): N/A

--- check: package_management
INFO: has conda-meta: False
INFO: $VIRTUAL_ENV: '/Users/user/.venv'

--- check: installed_packages
INFO: installed: tensorboard==2.13.0
INFO: installed: tensorflow==2.13.0rc1
INFO: installed: tensorflow-estimator==2.13.0rc0
INFO: installed: tensorboard-data-server==0.7.1

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.13.0'

--- check: tensorflow_python_version
INFO: tensorflow.__version__: '2.13.0-rc1'
INFO: tensorflow.__git_version__: 'v2.13.0-rc0-26-g57633696be6'

--- check: tensorboard_data_server_version
INFO: data server binary: None

--- check: tensorboard_binary_path
INFO: which tensorboard: b'/Users/user/.venv/bin/tensorboard\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 1024>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 1024>
Loopback infos: [(<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 0)), (<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 0, 0, 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET6: 30>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('0.0.0.0', 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'FVFF77UXQ05N'

--- check: stat_tensorboardinfo
INFO: directory: /var/folders/v2/bbhz14h10h31_42ks8bbnrlm0000gn/T/.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=79525584, st_dev=16777234, st_nlink=2, st_uid=501, st_gid=20, st_size=64, st_atime=1692350991, st_mtime=1692352818, st_ctime=1692352818)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['/Users/user/.venv/lib/python3.9/site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==1.2.0
appnope==0.1.2
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asttokens==2.0.5
astunparse==1.6.3
attrs==21.4.0
backcall==0.2.0
bcrypt==4.0.1
beautifulsoup4==4.10.0
black==23.1.0
bleach==4.1.0
boto3==1.26.60
botocore==1.29.60
cachetools==5.2.0
certifi==2022.6.15
cffi==1.15.0
charset-normalizer==2.1.0
click==8.1.3
cryptography==39.0.0
cycler==0.11.0
debugpy==1.6.0
decorator==5.1.1
defusedxml==0.7.1
entrypoints==0.4
executing==0.8.3
flatbuffers==23.5.26
fonttools==4.31.2
gast==0.4.0
google-auth==2.20.0
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
grpcio==1.54.2
gviz-api==1.10.0
h5py==3.8.0
idna==3.3
importlib-metadata==4.12.0
ipykernel==6.10.0
ipympl==0.9.3
ipython==8.2.0
ipython-genutils==0.2.0
ipywidgets==7.7.0
jedi==0.18.1
Jinja2==3.1.1
jmespath==1.0.1
joblib==1.2.0
jsonschema==4.4.0
jupyter==1.0.0
jupyter-client==7.2.0
jupyter-console==6.4.3
jupyter-core==4.9.2
jupyterlab-pygments==0.1.2
jupyterlab-widgets==1.1.0
keras==2.13.1rc0
kiwisolver==1.4.2
libclang==16.0.0
Markdown==3.4.1
MarkupSafe==2.1.1
matplotlib==3.5.1
matplotlib-inline==0.1.3
mistune==0.8.4
mpld3==0.5.9
mypy-extensions==1.0.0
nbclient==0.5.13
nbconvert==6.4.5
nbformat==5.2.0
nest-asyncio==1.5.4
notebook==6.4.10
numpy==1.22.3
oauthlib==3.2.0
opt-einsum==3.3.0
packaging==23.0
pandas==1.4.1
pandoc==2.1
pandocfilters==1.5.0
paramiko==3.0.0
parso==0.8.3
pathspec==0.11.0
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.0.1
pip==22.2.1
platformdirs==3.0.0
plotly==5.13.0
plumbum==1.7.2
ply==3.11
prometheus-client==0.13.1
prompt-toolkit==3.0.28
protobuf==3.20.3
psutil==5.9.0
ptyprocess==0.7.0
pure-eval==0.2.2
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycparser==2.21
Pygments==2.11.2
PyNaCl==1.5.0
pyparsing==3.0.7
pyrsistent==0.18.1
python-dateutil==2.8.2
pytz==2022.1
pyzmq==22.3.0
qtconsole==5.3.0
QtPy==2.0.1
requests==2.28.1
requests-oauthlib==1.3.1
rsa==4.9
s3transfer==0.6.0
scikit-learn==1.2.2
scipy==1.10.1
Send2Trash==1.8.0
setuptools==61.0.0
six==1.16.0
soupsieve==2.3.1
stack-data==0.2.0
tabulate==0.8.10
tenacity==8.2.1
tensorboard==2.13.0
tensorboard-data-server==0.7.1
tensorboard-plugin-profile==2.8.0
tensorboard-plugin-wit==1.8.1
tensorflow==2.13.0rc1
tensorflow-estimator==2.13.0rc0
tensorflow-macos==2.13.0rc1
termcolor==2.3.0
terminado==0.13.3
testpath==0.6.0
threadpoolctl==3.1.0
tomli==2.0.1
tornado==6.1
traitlets==5.1.1
typing_extensions==4.4.0
urllib3==1.26.11
wcwidth==0.2.5
webencodings==0.5.1
Werkzeug==2.2.1
wheel==0.37.1
widgetsnbextension==3.6.0
wrapt==1.15.0
zipp==3.8.1

``````

</details>

### Next steps

No action items identified. Please copy ALL of the above output,
including the lines containing only backticks, into your GitHub issue
or comment. Be sure to redact any sensitive information.

https://raw.githubusercontent.com/tensorflow/tensorboard/master/tensorboard/tools/diagnose_tensorboard.py

For browser-related issues, please additionally specify:

  - Browser type and version (e.g., Chrome 64.0.3282.140):
   Version 115.0.5790.170 (Official Build) (arm64)

  - Screenshot, if it’s a visual issue:
![Screenshot 2023-08-18 at 11 07 11](https://github.com/tensorflow/tensorboard/assets/65665931/27a2ed8f-85bb-4bb4-baa1-8beb7b25c88e)

## Issue description

When I click the Export as csv button in the GUI (See screenshot above), nothing happens. It does not download the file. I have tried Firefox as well.

**Comment by rileyajones** [comment]
This looks like an issue with the TensorFlow profiler plugin which is part of a different repo and maintained by different people
https://github.com/tensorflow/profiler

**Comment by cfRod** [comment]
Is it possible to get the csv directly without the GUI?
